cancer/rank_model/cancer_rank_model_-v1.2-.ini

```
@@ -0,0 +1,454 @@
+[Version]
+  version = 1.1
+  name = rank_model
```

What do you think  about starting with a version for this? I think we can also name it more specifically.
```suggestion
  version = 0.1
  name = general_rank_model_cancer
```